src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -348,6 +447,9 @@ protected AIState attackProtect()
         return GUARD_ATTACK_PHYSICAL;
     }
 
+    /**
+     * @return
```

Comment:
Also, add content to javadoc

Reply:
Yeah, but if you add the javadoc comment it would be good to fill it in (what does the method do, what does it return)